ale/drivers/messenger_drivers.py

```diff
@@ -172,10 +186,10 @@ def instrument_id(self):
         : str
           instrument id
         """
-        return self.id_lookup[self.label['IsisCube']['Instrument']['InstrumentId']]
+        return self.id_lookup[super().instrument_id]
```

Comment:
This id_lookup appears in both the isis label and pds label drivers. Suggest moving it above the top "class" so it can be referenced by both without having two identical copies, and make id_lookup all caps. Thanks @jlaura

Reply:
Does my latest commit work? Now that I'm thinking about it kind of seems like it should be declared as a function instead of a variable if its going to be outside of the classes right?